src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/AbstractFlowPass.vb

```diff
@@ -1542,7 +1552,7 @@ lUnsplitAndFinish:
 
         Private Function VisitObjectInitializerExpressionBase(node As BoundObjectInitializerExpressionBase) As BoundNode
             For Each initializer In node.Initializers
-                Visit(initializer)
```

Comment:
This line is causing an incorrect state when ```initializer``` is a logical expression, which can happen in presence of errors.